README.md

```diff
@@ -26,7 +31,7 @@ default: `false`
 
 ### `debug`
 
-write debug informations to stdout
+write debug information to stdout
```

Comment:
i would like to use `informations` because debug prints skips, statistics etc

Reply:
information is uncountable, so it has no plural. It's often misused. That's why I fixed it.

https://dictionary.cambridge.org/grammar/british-grammar/information

But I can rollback, if you prefer.

Please tell me.